kythe/rust/indexer/src/providers.rs

```diff
@@ -91,26 +115,28 @@ impl KzipFileProvider {
 impl FileProvider for KzipFileProvider {
     /// Given a file hash, returns whether the file exists in the kzip.
     fn exists(&mut self, file_hash: &str) -> bool {
-        let name = format!("files/{}", file_hash);
+        // root_name contains a trailing forward-slash, so it's needed in the format
+        // string.
+        let name = format!("{}files/{}", self.root_name, file_hash);
         self.zip_archive.by_name(&name).is_ok()
     }
 
-    /// Given a file hash, returns the string contents of the file from the
+    /// Given a file hash, returns the vect of bytes of the file from the
```

Comment:
Feel free to spell out vector. :)  Helps with code search later too.

Reply:
Oh, I believe this was a typo